website/static/images/users/computerbase.svg

```diff
@@ -0,0 +1,18 @@
+		<path fill="#fff" d="M39.73,3.4c-1.92-1.11-4.11-1.97-6.5-2.54C30.85,0.29,28.24,0,25.48,0c-3.45,0-6.8,0.66-9.93,1.95C12.4,3.24,9.67,5.06,7.42,7.36c-2.3,2.37-4.13,5.21-5.44,8.45C0.67,19.06,0,22.71,0,26.69c0,5.11,1.54,9.1,4.57,11.85c3.01,2.75,7.32,4.14,12.81,4.14c2.01,0,3.61-0.1,4.75-0.3c1.14-0.2,2.37-0.5,3.64-0.87c0.99-0.29,2.05-0.67,3.13-1.14c1.06-0.45,1.99-0.84,2.78-1.16c0.18-0.07,0.32-0.23,0.36-0.42l2.32-9.54c0.04-0.18,0-0.37-0.11-0.52c-0.11-0.15-0.29-0.23-0.48-0.23h-1.06c-0.14,0-0.27,0.05-0.38,0.13c-1.87,1.5-3.88,2.75-5.97,3.71c-2.06,0.95-4.3,1.43-6.64,1.43c-2.55,0-4.51-0.66-5.8-1.97c-1.3-1.3-1.95-3.49-1.95-6.49c0-4.78,1.25-8.77,3.71-11.84C18.15,10.4,21.2,8.91,25,8.91c2.15,0,4.04,0.41,5.61,1.22c1.6,0.82,3.24,2.13,4.87,3.89c0.11,0.12,0.27,0.19,0.44,0.19h1.17c0.28,0,0.52-0.19,0.59-0.46l2.34-9.68C40.08,3.81,39.96,3.54,39.73,3.4z"/>
+		<path fill="#fff" d="M55.92,42.23c2.15-0.82,3.81-1.92,4.97-3.31c1.41-1.69,2.13-3.84,2.13-6.41c0-3.07-1.37-5.49-4.07-7.18c-2.61-1.63-6.4-2.46-11.27-2.46H33.21c-0.28,0-0.53,0.2-0.59,0.47l-1.01,4.39c0.03-0.02,0.06-0.04,0.09-0.07c0.32-0.26,0.72-0.4,1.13-0.4h1.06c0.56,0,1.08,0.25,1.43,0.7c0.35,0.44,0.47,1.01,0.33,1.56l-2.32,9.54c-0.14,0.57-0.54,1.04-1.09,1.26c-0.77,0.31-1.68,0.69-2.76,1.15c-0.38,0.16-0.76,0.31-1.13,0.46l-4.88,21.31c-0.04,0.18,0,0.37,0.12,0.52S23.88,64,24.06,64h17.17c6.34,0,11.34-1.23,14.84-3.64c3.58-2.47,5.39-5.95,5.39-10.34c0-2.3-0.62-4.11-1.83-5.4C58.72,43.64,57.47,42.84,55.92,42.23z M47.72,54.08C46.21,55,44,55.47,41.15,55.47h-4.4l2-8.73h4.87c2.24,0,3.88,0.3,4.88,0.89c0.91,0.54,1.35,1.41,1.35,2.65C49.85,51.96,49.15,53.21,47.72,54.08z M49.38,37.44c-1.14,0.79-2.78,1.19-4.88,1.19H40.6l1.66-7.23h4.12c1.52,0,2.71,0.24,3.53,0.71c0.73,0.42,1.08,1.07,1.08,2.03C50.99,35.61,50.46,36.69,49.38,37.44z"/>
+		<path fill="#fff" d="M54.65,18.7c0-1.18-0.17-2.25-0.5-3.22c-0.33-0.96-0.8-1.78-1.42-2.47c-0.62-0.68-1.37-1.21-2.26-1.58c-0.89-0.37-1.9-0.55-3.02-0.55c-1.12,0-2.13,0.19-3.02,0.57c-0.89,0.38-1.65,0.91-2.27,1.59c-0.62,0.68-1.1,1.51-1.43,2.48c-0.33,0.97-0.5,2.04-0.5,3.2c0,1.08,0.13,2.05,0.37,2.92h4.58c-0.25-0.71-0.37-1.69-0.37-2.92c0-1.61,0.22-2.77,0.66-3.49c0.44-0.72,1.1-1.08,1.98-1.08c0.88,0,1.54,0.36,1.97,1.08c0.43,0.72,0.65,1.88,0.65,3.47c0,1.29-0.13,2.29-0.4,3.01c0.04,0,0.08,0.01,0.13,0.01c1.56,0.04,3,0.17,4.32,0.38C54.47,21.12,54.65,19.99,54.65,18.7z"/>
+		<path fill="#fff" d="M76.08,12.04c-0.63-0.77-1.6-1.15-2.91-1.15c-0.75,0-1.5,0.18-2.23,0.54c-0.73,0.36-1.43,0.91-2.09,1.65c-0.57-1.46-1.73-2.19-3.47-2.19c-0.77,0-1.51,0.17-2.2,0.53s-1.37,0.91-2.01,1.66l-0.33-1.69H57v11.41c1.15,0.37,2.18,0.82,3.07,1.38c0.5,0.31,0.93,0.66,1.34,1.02v-9.31c0.35-0.54,0.72-0.96,1.1-1.26c0.38-0.3,0.76-0.46,1.13-0.46c0.4,0,0.7,0.12,0.9,0.37c0.19,0.25,0.29,0.68,0.29,1.29v10.29h4.41V15.88c0.33-0.54,0.69-0.96,1.08-1.26s0.77-0.46,1.16-0.46c0.4,0,0.7,0.13,0.88,0.39c0.18,0.26,0.28,0.71,0.28,1.36v10.21h4.41V15.29C77.03,13.89,76.71,12.8,76.08,12.04z"/>
+	</g>
+</svg>
```

Comment:
Since you are submitting the svg, could you please run the `yarn svgo static/images/users/computerbase.svg` from the `website` directory?

https://github.com/prettier/prettier/blob/master/website/package.json#L7